libs/ngx-dsfr/checkbox/src/lib/checkbox.component.ts

```diff
@@ -44,6 +44,7 @@ export class DsfrCheckboxComponent implements ControlValueAccessor, OnInit, OnCh
   @Input() legend = '';
   @Input() fieldSetId = '';
   @Input() hint = '';
+  @Input() name = '';
```

Comment:
Great 👍
Now, if I'm not mistaken, you can remove the `name` property from the `CheckboxItem` interface on line 26...

Reply:
Done